src/main.rs

```diff
@@ -127,6 +127,10 @@ fn build_hyperfine_options(matches: &ArgMatches<'_>) -> Result<HyperfineOptions,
         (None, None) => {}
     };
 
+    options.names = matches
+        .values_of("command-name")
+        .map(|values| values.map(String::from).collect::<Vec<String>>());
```

Comment:
I think we should show an error message if there are *more* names than commands:
```
hyperfine 'echo a' -n a -n b
```

Having too few is okay, but probably not really intended in most cases.

Reply:
OK I now return an error in that case, see https://github.com/sharkdp/hyperfine/pull/330/commits/9ed5f0de40b295b5b1a670d62f439ea8dcbe98cc